pkgs/tools/misc/killport/default.nix

```diff
@@ -0,0 +1,25 @@
+{ rustPlatform
+, lib
+, fetchCrate
+, llvmPackages
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "killport";
+  version = "0.7.0";
+
+  src = fetchCrate {
+    inherit pname version;
+    hash = "sha256-sPmHrq/VVSwgUczNdDkgx/+aSETjry4WJ6YfIeIrjGU=";
+  };
+
+  cargoSha256 = "sha256-u5HJLBT8hYKaJQBFaJUb2yyQrmPsN6+P0mTH+iHDPu0=";
+
+  LIBCLANG_PATH = "${llvmPackages.libclang.lib}/lib";
```

Comment:
```suggestion
  nativeBuildInputs = [ rustPlatform.bindgenHook ];
```

Reply:
> This suggestion seems to break build on `x86_64-darwin`

Yes, sorry about this. I guess the global SDK bump is really an urgent need on that platform.